RxCocoa/Runtime/include/_RXObjCRuntime.h

```diff
@@ -98,5 +100,3 @@ BOOL RX_is_method_signature_void(NSMethodSignature * __nonnull methodSignature);
 
 /// Default value for `RXInterceptionObserver.targetImplementation`.
 IMP __nonnull RX_default_target_implementation(void);
```

Comment:
I don't think this needs to be included, right?

Reply:
I think that we just need to expose 

```
/// Returns `YES` in case method has `void` return type.
BOOL RX_is_method_with_description_void(struct objc_method_description method);
 /// Extracts arguments for `invocation`.
NSArray * __nonnull RX_extract_arguments(NSInvocation * __nonnull invocation);
 /// Returns `YES` in case methodSignature has `void` return type.
BOOL RX_is_method_signature_void(NSMethodSignature * __nonnull methodSignature);
```